src/Microsoft.AspNet.Mvc.Core/Resources.resx

```diff
@@ -418,4 +418,7 @@
   <data name="ResponseCache_SpecifyDuration" xml:space="preserve">
     <value>If the '{0}' property is not set to true, '{1}' property must be specified.</value>
   </data>
+  <data name="FormatterMappings_NotValidMediaType" xml:space="preserve">
+    <value>The media type {0} is not valid. The media type containing "&lt;mediatype&gt;/*" are not supported.</value>
```

Comment:
> The media type containing "&lt;mediatype&gt;/*" are not supported.

This is a little ambiguous and not the whole truth - wildcards are not permitted on the `type` or `subtype`. I'd say something like:

> MediaTypes containing wildcards (*) are not allowed in formatter mappings.

Also quotes around `{0}` we want to clearly delimit the user input